component/src/test/resources/testng.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Siddhi-IO-GCS-Test-Suite">
+    <test name="Siddhi-io-google cloud storage-tests" enabled="true" preserve-order="true" parallel="false">
+        <classes>
+<!--            <class name="io.siddhi.extension.io.gcs.sink.TestCaseOfGCSSink"/>-->
```

Comment:
Tests are disabled

Reply:
Can't run the test as it requires to establish a connection with the GCS Service for that we need to have credentials